src/sentry/static/sentry/app/components/events/eventMessage.jsx

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import ErrorLevel from 'app/components/events/errorLevel';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+
+const Message = styled('span')`
+  ${overflowEllipsis}
+  width: auto;
+  max-height: 38px;
+`;
+
+function EventMessage({className, level, levelIndicatorSize, message, annotations}) {
+  return (
+    <div className={className}>
+      <StyledErrorLevel size={levelIndicatorSize} level={level}>
+        {level}
+      </StyledErrorLevel>
+      {message && <Message>{message}</Message>}
+      {annotations}
+    </div>
+  );
+}
+
+EventMessage.propTypes = {
+  level: PropTypes.oneOf(['error', 'fatal', 'info', 'warning', 'sample']),
```

Comment:
Required?

Reply:
It's optional (updated component to reflect it)